src/main/java/com/parsing/parsers/prozorro/URLListBuilder.java

```diff
@@ -10,31 +11,37 @@
 @Slf4j
 public class URLListBuilder {
 
-    private static final LocalDate START_DATE = LocalDate.of(2023, 02, 21);
-    private static final LocalDate END_DATE = LocalDate.of(2023, 02, 21);
+    private static final LocalDate START_DATE = LocalDate.of(2023, 04, 04);
```

Comment:
should we provide these dates from controller or from the properties file?

Reply:
Done.